azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositePushNotification.java

```diff
@@ -20,12 +20,12 @@ public class CallCompositePushNotificationInfo {
     private final String eventTypeResult;
 
     /**
-     * Create {@link CallCompositePushNotificationInfo}.
-     * @param notificationInfoMap Notification info map.
+     * Create {@link CallCompositePushNotification}.
+     * @param notification Notification info map.
      */
-    public CallCompositePushNotificationInfo(final Map<String, String> notificationInfoMap) {
+    public CallCompositePushNotification(final Map<String, String> notification) {
         final com.azure.android.communication.calling.PushNotificationInfo pushNotificationInfo =
-                com.azure.android.communication.calling.PushNotificationInfo.fromMap(notificationInfoMap);
+                com.azure.android.communication.calling.PushNotificationInfo.fromMap(notification);
```

Comment:
no import here? fully resolved?

Reply:
Hi Adam, this is for MAUI. We do not want to have native SDK import in public API. For native SDK because of retrofuture we can not generate bindings, thus having native SDK public API will cause UI bindings to fail. 
+ When testing on cross platform, if we see issue we may need to use reflection's.